packages/manager/apps/web-domains/src/alldoms/components/ServiceDetail/ServiceDetailSubscribing/ServiceDetailSubscribing.component.tsx

```diff
@@ -59,6 +59,7 @@ export default function ServiceDetailSubscribing({
           terminateUrl={'terminate'}
           lifecycleCapacities={lifecycleCapacities}
           whichAction={ActionEnum.OnlyRenew}
+          cancelTerminateUrl={'terminate/cancel'}
```

Comment:
Don't use the URL directly in the code, use const to make sure that if we need to update the path, it will be easier to change.